### ISSUE TITLE
Handle live broadcasts (Youtube/Facebook)

### ISSUE THREAD

**Issue by vheuken** [thibaudgg/video_info#152]
This has been discussed in #151 with @Brotakuu

* `.live_stream?` function that returns whether or not the video is a live broadcast
* `.live_status` function that returns a symbol reflecting the status or nil if it is not a live stream. Potential return values are `:live`, `:upcoming`, and `:ended`, though we should look into the info we can get from Facebook and Youtube first.
* Need to figure out how to test, since a broadcast can't be live or upcoming forever.

**Comment by Brotakuu** [comment]
Looks like facebook similarly has a `live_status` field that returns `enum` status of the video.

https://developers.facebook.com/docs/graph-api/reference/video/

**Comment by vheuken** [comment]
Right. We'd need to find out exactly what it returns. I'll try to play with it tonight while I'm working on FB support.
